server/node_test.go

```diff
@@ -42,7 +42,7 @@ import (
 // of engines. The server and node are returned. If gossipBS is not
 // nil, the gossip bootstrap address is set to gossipBS.
 func createTestNode(addr net.Addr, engines []engine.Engine, gossipBS net.Addr, t *testing.T) (
-	*rpc.Server, *Node) {
+	*rpc.Server, *Node, error) {
```

Comment:
I dislike this change as it just adds a lot of extra lines of test code without adding useful functionality. If something is an unrecoverable and fully-unexpected error in a test, we should just t.Fatal(err) as early as possible. The previous code was much simpler.